middlewares/validateJWT.js

```diff
@@ -0,0 +1,30 @@
+const moment = require('moment');
+const jwt = require("jsonwebtoken");
+
+const ensureAuthenticated = (req, res, next) => {
+    if(!req.headers.authorization) {
+        return res.status(403).json({
+            result: null,
+            message: "Your request doesn't have authorization header"
+        });
+    }
+
+    const token = req.headers.authorization.split(" ")[1];
```

Comment:
Validemos tambien que exista el texto `Bearer` 
`  const [ bearer, token ] = authorization.split(' ');

    if (bearer !== 'Bearer') {
        // devolver error Not authorized
    }
`